frontend/src/common/styles/globals.css

```diff
@@ -153,9 +153,8 @@ input.form-control {
 #navbar-main .nav-link {
   color: white;
   font-size: large;
-  padding: 1rem;
+  padding: 1rem 0.5rem; /* Adjusted padding value */
```

Comment:
sweep: Try running the app in your branch `sweep/navbar-css-fix` and take a screenshot of the navbar. Send it in the PR thread